ui-spacetimechart/src/stories/layers.stories.tsx

```diff
@@ -30,6 +30,51 @@ const CONFLICTS = [
   },
 ];
 
+const OCCUPANCY_BLOCKS = [
+  {
+    timeStart: +START_DATE + 42 * MINUTE,
+    timeEnd: +START_DATE + 43 * MINUTE,
+    spaceStart: 10 * KILOMETER,
+    spaceEnd: 14 * KILOMETER,
+    color: '#CAEDDB',
```

Comment:
so no explicit color name?

Reply:
Yes as it is repeated several time, it should be in variable for maintainability, even if it's a story, it's an example of implemntation that we or anybody can use

Reply:
```
signal_updates: {
   ...
   The color of the aspect (Bits 24-31 are alpha, 16-23 are red, 8-15 are green, 0-7 are blue) */
    color: number;
   ...
  }[];
```
It will be only useful for this repo since colors don't have explicit names returned by the back either. But if it improves the clarity of the code I can add it.

Reply:
How should I name them ? 
- occupied : SEMAPHORE
- free : ?
- warning : ?

Reply:
![image](https://github.com/user-attachments/assets/151eb6c2-3db0-42e3-9161-38d333e0f7af)

Reply:
done :)